lib/src/exceptions/name/invalid_reserved_name_exception.dart

```diff
@@ -36,7 +18,7 @@ class InvalidReservedNameException implements Exception {
   late final String _message;
 
   /// constructor for InvalidReservedNameException,
```

Comment:
Documentation for functions and constructors should describe what it *does*, not what it *is*.  

E.g.
```dart
/// Constructs an exception indicating that a reserved name is not valid.
///
/// The [message] explains the reasoning behind this exception being thrown.
```

Reply:
Alright, updated the doc.